content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: 'setUTCFullYear()'
+Description: 'setUTCFullYear() method of Date instances'
```

Comment:
```suggestion
Description: 'Used to set the year value of a Date object in UTC (Coordinated Universal Time)'
```
Description should be small and descriptive explanation of what the function will does.